amendments.json

```
@@ -148,6 +148,14 @@
       "type": "correction",
       "status": "candidate",
       "id": 13
+    },
+    {
+      "description": "Allow remote offer rid pruning of encodings through the client answer.",
+      "pr": 2758,
+      "difftype": "modify",
+      "type": "correction",
+      "status": "candidate",
+      "id": 19
```

```suggestion
      "id": 22
```
19 has been used by #2785